.github/workflows/add-to-project-issues.yml

```diff
@@ -0,0 +1,14 @@
+name: "Project triage: Issues"
+
+on:
+  issues:
+    types: [opened, transferred]
+
+jobs:
+  add-to-project-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@main
+        with:
+          project-url: https://github.com/orgs/compilerla/projects/${{ secrets.GH_PROJECT }}
+          github-token: ${{ secrets.GH_PROJECTS_TOKEN }}
```

Comment:
@lalver1 I think to create this, you'd log in to GitHub as @compiler-bot and generate a new PAT. Then you can add that as a GitHub secret. 

I think an organization-level secret makes sense? cc @thekaveman

Reply:
I see, thanks @angela-tran!

Reply:
Thanks @angela-tran!

Org secret definitely makes sense for the PAT generated from the @compiler-bot for `GH_PROJECTS_TOKEN`. That's how we do it in Cal-ITP. That org secret should be restricted and only available to certain repos (those where we will auto-add issues to the project).

I think for the `GH_PROJECT` (i.e. the ID of the project) we can make that an org variable, no need for it to be secret.